### DISCUSSION TITLE
Proposed Virtual Worlds and Simulations meeting agenda for 2023-02-22

### DISCUSSION THREAD

**Discussion by OBWANDO** [Open-MV/fig-virtualworldsim#6] [Meetings]
## Meeting Details

- **Date/Time:** February 22th, 2023 @ Wednesday 12:00 PM ET / 17:00 UTC / 6:00 PM CET
- **Location:** Link should be posted in the `#Virtual-world-sim` text and voice channel on Discord at least 5 days before the meeting.
- **Moderator:** @LaylBongers 
- **Note Taker** 

The [FIG Virtual Worlds and Simulations Meetings](https://github.com/Open-MV/fig-virtualworldsim/tree/main/meetings) repo contains the history past meetings, including a link to the agenda, recording, notes, and resources.

## Meeting Agenda

- Add tracking issues for database items that we consider at this point to be obvious high priorities to be handled soon, and expand on the two already added.
Followup on:
- Priority item on "metaverse viewer project" (Tracking issue https://github.com/Open-MV/fig-virtualworldsim/issues/4), we need to start looking at this and exploring what existing projects we can make use of too to reduce our workload in this prototyping stage.
- Break down the "Avatar-as-Metaobject" task into its dependencies, and derived features, so we can fit it under a larger "User Avatars" epic.
- Database Feature triage

## Action Items

**Create actionable items from proposed topics**

## Items for next week's meeting
**List items that need to be discussed next week**

## Open Discussion Items

List any additional items below!

**Comment by LaylBongers** [comment]
Some quick additional discussion topics, if there's time:
- Daicon has been split out from the dacti project, and is now open for feedback and contributions. Is this the right time to move it out of the sandbox group?
- Recommended in-world fonts for early prototypes. Readability in fonts at different angles and small sizes will quickly become an issue if prototypes get traction. (consider humanist options like "Open sans" and "Merriweather Sans", or even "Atkinson Hyperlegible") Additionally, a chosen option should have a broad character set support for languages.

**Comment by LaylBongers** [comment]
## Outcomes from Discussion topics
- specific font doesn't matter too much partially because there's big disagreement, but we could suggest at least using a humanist font like Open Sans

## Action Items
- take a look at daicon if there's time and interest
- clarify use cases of daicon, potentially with user stories
- reach out to projects and see what we can do for them
